incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
> Was there a particular concern you had?

My main concern is that keeping the Dockerfile in the chart is not any more transparent than having it in a separate repository. However, if the repository had a CI in place to build the image, then I have some visibility into what has been pushed and when (we should consider making this a requirement in a separate discussion). I'm also not sure we should encourage creating Docker images specifically for use in Helm charts, but maybe in cases like this where it's only a small tweak, it's ok.

> I think aspirationally we should have a process in the CI pipeline to auto build/push images. Maybe with a convention on where images would live (images folder) and that the tag would be equivalent to the chart version.

I think this could be an interesting direction, though of course we will need to think about how we reference images (presumably the CI will need to build and push before testing the chart). We've discussed packing images with charts and things like embedding a registry with the images used in the chart within Helm, so there may be different avenues.

All that said, I don't think it's worth holding up the PR on this, but I think it would be good to bring this up for discussion in a future Helm/SIG apps meeting.